app/src/main/res/layout-sw600dp/administrator_controls_activity.xml

```diff
@@ -12,12 +12,18 @@
       android:layout_height="match_parent"
       android:background="@color/oppia_grey_background">
 
-      <include
-        android:id="@+id/administrator_controls_activity_toolbar"
-        layout="@layout/toolbar"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        <androidx.appcompat.widget.Toolbar
```

Comment:
Here & elsewhere in the PR: the indentation doesn't match anymore. Please reformat this file using Android Studio's code reformat. Ditto for the other changed layouts in this PR.

Reply:
All affected XML layout files have been reformated. PTAL